packages/cli-hooks/src/check-update.js

```diff
@@ -1,13 +1,25 @@
 #!/usr/bin/env node
```

Comment:
I think we can drop these now, yeah?

Reply:
I hoped so but think it might be necessary to run this script as a standalone script 🤔 This is what happens when I try removing it for `get-hooks` in a sample app:

```sh
$ hermes manifest
Check /Users/ethan.zimbelman/.slack/logs/slack-debug-20240126.log for full error logs

🚫 There was an error while reading the Slack Configuration file (`slack.json`) or running the `get-hooks` hook (sdk_config_load_error)

Error Details:

1: Command for '' returned an error: exit status 2
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 1: import: command not found
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 2: import: command not found
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 4: /Applications: is a directory
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 5: LICENSE: command not found
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 6: LICENSE: command not found
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 7: node_modules/: is a directory
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 9: syntax error near unexpected token `process.argv[1]'
/private/tmp/sample_app/node_modules/.bin/slack-cli-get-hooks: line 9: `if (fs.realpathSync(process.argv[1]) === fileURLToPath(import.meta.url)) {' (sdk_hook_invocation_failed)
Suggestion: Run `slack doctor` to check that your system dependencies are up-to-date.
```

Confusingly, both of the following commands work fine without `#!/usr/bin/env node`:

```sh
$ node ./node_modules/.bin/slack-cli-get-hooks
$ sh -c "node ./node_modules/.bin/slack-cli-get-hooks"
```

This kinda makes me think something strange might be happening with how the CLI is calling these.

I also just noticed this command returns the same `import: command not found`+ errors:

```sh
$ npx slack-cli-get-hooks
```

I think that's a solid hint, but I'm not sure if we want to call the hooks another way? I'll poke into this a bit more soon!

Reply:
Ah but of course, this is the command we use in the `slack.json`:

```json
{
  "hooks": {
    "get-hooks": "npx -q --no-install -p @slack/cli-hooks slack-cli-get-hooks"
  }
}
```

I'm leaning towards keeping this command in the as is in the `slack.json` since `npx` takes care of the symlink magic, but substituting it for calling `node` might remove the need for `#!` 🤔

Reply:
All good, I'm surprised by the behaviour too. Oh well. Not a big deal.